dags/datasets-downstream1.py

```diff
@@ -0,0 +1,26 @@
+import pendulum
+
+from airflow import DAG, Dataset
+from airflow.operators.bash import BashOperator
+
+"""
+This is an example of a DAG that consumes two datasets.
+The DAG only runs once dag1_dataset and dag2_dataset have been updated.
+"""
+
+dag1_dataset = Dataset('s3://dataset1/output_1.txt')
+dag2_dataset = Dataset('s3://dataset2/output_2.txt')
+
+with DAG(
+    dag_id='dataset_downstream_1_2',
+    catchup=False,
+    start_date=pendulum.datetime(2021, 1, 1, tz="UTC"),
+    schedule=[dag1_dataset, dag2_dataset],
+    tags=['downstream'],
+) as dag3:
```

Comment:
```suggestion
) as dag:
```

Very nitpicky detail but I can see my beginner self being confused thinking it has to be dag3 because it is waiting for datasets 1 and 2 or some other weird idea 😅